examples/querying-with-streams/querying_with_streams.md

```diff
@@ -1,10 +1,6 @@
 # Querying with streams
 
-If stream terminates with `error`, result of `query expression` is an `error`. You cannot use `foreach`
-on `stream` type with termination type that allows `error`. Instead use `from` with `do` clause; the
-result is a subtype of `error?`. Use `stream` keyword in front of `from` to create a `stream` which is
-lazily evaluated. The failure of `check` within the `query` will cause the `stream` to produce an
-`error` termination value.
+If stream terminates with `error`, the result of the query expression is an `error`. You cannot use `foreach` on `stream` type with termination type that allows `error`. Instead, use `from` with `do` clause, where the result is a subtype of `error?`. Use `stream` keyword in front of `from` to create a `stream` which is lazily evaluated. The failure of `check` within the query will cause the `stream` to produce an `error` termination value.
```

Comment:
```suggestion
If a stream terminates with `error`, the result of the query expression is an `error`. You cannot use `foreach` on a `stream` type with a termination type that allows `error`. Instead, use `from` with the `do` clause where the result is a subtype of `error?`. Use the `stream` keyword in front of `from` to create a `stream`, which is lazily evaluated. The failure of `check` within the query will cause the `stream` to produce an `error` termination value.
```
Is it fine to use the word `lazily` here? Or, do we have a more formal alternative word?